docs/Model_Assumptions.rst

```diff
@@ -139,6 +138,34 @@ Also, if you are aiming at very high minimal renewable factors, the simulation t
 
 The minimum renewable share is introduced to the energy system by `D2.constraint_minimal_renewable_share()` and a validation test is performed with `E4.minimal_renewable_share_test()`.
 
+Minimal degree of autonomcy constraint
+######################################
+
+The minimal degree of autonomy constraint requires the capacity and dispatch optimization of the MVS to reach at least the minimal degree of autonomy defined within the constraint. The degree of autonomy of the optimized energy system may also be higher than the minimal degree of autonomy. Please find the definition of here: :ref:`_kpi_degree_of_autonomy`
+
+The minimal degree of autonomy is applied to the whole, sector-coupled energy system, but not to specific sectors. As such, energy carrier weighting plays a role and may lead to unexpected results. The constraint reads as follows:
+
+.. math::
+        minimal degree of autonomy <= \frac{\sum local demand \cdot weighting factor - \sum consumtion from energy providers}{\sum local demand  \cdot weighting factor}
+
+:Deactivating the constraint:
+
+The minimal degree of autonomy constraint is deactivated by inserting the following row in `constraints.csv` as follows:
+
+```minimal_degree_of_autonomy,factor,0```
+
+:Activating the constraint:
+
+The constraint is enabled when the value of the minimal renewable factor factor is above 0 in `constraints.csv`:
```

Comment:
The highest accepted value is 1 which optimizes the system in a fully autonomous way.

Reply:
Constraint is included in csv